variants/aws-dev/Cargo.toml

```diff
@@ -12,7 +12,6 @@ grub-set-private-var = true
 unified-cgroup-hierarchy = true
 xfs-data-partition = true
 uefi-secure-boot = true
-systemd-networkd = true
 
```

Comment:
So if I understand correctly, we are shifting the dependency to default into `systemd-networkd`? And we rely upon the netdog-wicked to bring in wicked.

Reply:
correct